setup.py

```diff
@@ -134,7 +134,9 @@ def initialize_options(self) -> None:
 )
 
 airflow_dependencies = [
-    'apache-airflow>=1.10,<2'
+    # Minimum version here is needed to avoid syntax error in setup.py
```

Comment:
This file, one of the 3 largest in the project, increased in size to 379 lines.  The total size of those files is now 3421 lines (target: 1149).  Is this file complex enough to refactor?